programming-languages.md

```diff
@@ -31,6 +31,35 @@ The set of languages in the core group will change over time. This is likely
 to be through the adoption of a new language into the core group, and with
 an existing language being put on hold for new work.
 
+## Code styleguides
+
+Developers read code much more often than they write it. These guidelines
+are intended to improve the readability of code and make it consistent
+across GDS projects.
+
+A style guide is about consistency. Consistency with this style guide is
+important. Consistency within a project is more important. Consistency within
+one module or function is most important.
+
+But most importantly: know when to be inconsistent -- sometimes the style guide
+just doesn't apply. When in doubt, use your best judgement. Look at other
+examples and decide what looks best. And don't hesitate to ask!
```

Comment:
If we're going to say "don't hesitate to ask", we need to say who people should be asking.